spec/services/sign_in/session_revoker_spec.rb

```diff
@@ -66,7 +66,7 @@
 
           context 'and client in session does not match an existing client configuration' do
             let(:expected_error) { ActiveRecord::RecordNotFound }
-            let(:expected_error_message) { "Couldn't find SignIn::ClientConfig" }
+            let(:expected_error_message) { /Couldn't find SignIn::ClientConfig/ }
```

Comment:
Error messages began appending more information beyond the message text, so we just used a regex on the expected text.

Reply:
can you put what the error message was here? I want to make sure the message change was benign

Reply:
The error messages are now including the explicit query being used to find the record:
`#<ActiveRecord::RecordNotFound "Couldn't find SignIn::ClientConfig with [WHERE \"client_configs\".\"client_id\" = $1]">` 🤷

Reply:
hmmm I guess that's fine